cpu/mips32r2_common/include/periph_cpu.h

```diff
@@ -0,0 +1,3 @@
+/* This file must exist to get timer code to build */
```

Comment:
mips32r2_common can be selected as a CPU on its own for testing on FPGA systems with minimal peripherals, this is what BOARD=mips-malta uses.